src/components/PlioQuestion.vue

```diff
@@ -90,7 +91,13 @@
 </template>
 
 <script>
+import LoadingSpinner from './LoadingSpinner.vue';
+
+// For how long does the spinner show (in milliseconds)
+var loadTime = 1500
```

Comment:
This time has been set because there was a bug occurring earlier.
Earlier : User would select answer, press submit, close button will show up, press close, and modal would close. 
BUG: If the close button was pressed instantly after pressing submit, the player would break, come out of fullscreen, throw an error and would pause (these things happened inconsistently). 

Solution: Apparently we needed to wait for some time. Hence the loading spinner on the question modal.